src/Workspaces/Core/Portable/Simplification/NamingStyleOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Diagnostics.Analyzers.NamingStyles;
+
+#if CODE_STYLE
+namespace Microsoft.CodeAnalysis.Internal.Options
+#else
+using Microsoft.CodeAnalysis.Options;
+namespace Microsoft.CodeAnalysis.Simplification
+#endif
+{
+#if CODE_STYLE
+    public static class NamingStyleOptions
+#else
+    internal static class NamingStyleOptions
+#endif
```

Comment:
this is just reall nasty to read.

Reply:
I know, I’ll see if I can work on the OptionDefinition based design that Sam was proposing to get rid of this Options related mess.